checklog-odoo.cfg

```diff
@@ -0,0 +1,3 @@
+[checklog-odoo]
+ignore=
+    WARNING.*0 failed, 0 error\(s\).*
```

Comment:
Why this addition ?

Reply:
This module originally had no test case and the new maintenance-tools test case was raising errors even with WARNING. So I added it to avoid being labeled as ERROR

Reply:
hum. something is wrong if CI is red if there are no test, it is a valid use case. I think that something should be fixed in checklog-odoo tools instead. 

@sbidoul : what do you think ?

Reply:
> something should be fixed in checklog-odoo tools instead.

We don't want to special case this in checklog-odoo, because we have had situations where for some reason tests did not run had that error was actually helpful.

This problem will only occur when the first module migrated has no test, so in practice this should not occur often, I guess.